src/robot/model/body.py

```diff
@@ -122,7 +130,7 @@ def create(self):
     def create_keyword(self, *args, **kwargs):
         return self._create(self.keyword_class, 'create_keyword', args, kwargs)
 
-    def _create(self, cls, name, args, kwargs):
+    def _create(self, cls:Union[Type[C], None], name: str, args, kwargs) -> C:
```

Comment:
- `cls` can only be `None` is there's a bug in our code so it shouldn't be included in type hints.
- This method creates body items so I believe this should be `_create(self, cls: Type[BodyItem], name: str, args: list, kwargs: dict) -> BodyItem`
- This is mostly internal code so I'm not sure are type hints needed in general. If the above works, it may be simple enough to be useful, but I believe more complex type hints with `TypeVar` aren't worth the effort.